mlbench_core/api.py

```diff
@@ -452,21 +447,28 @@ def create_run(
             "max_bandwidth": max_bandwidth,
             "gpu_enabled": "true" if gpu_enabled else "false",
             "light_target": "true" if light_target else "false",
+            "run_all_nodes": "true" if run_all_nodes else "false",
         }
 
         if custom_image_name is not None:
             data["image_name"] = "custom_image"
             data["custom_image_name"] = custom_image_name
             data["custom_image_command"] = custom_image_command
-            data["custom_image_all_nodes"] = custom_image_all_nodes
+            # data["custom_image_all_nodes"] = custom_image_all_nodes
```

Comment:
Delete this if it's not needed, no need to polute the repo with commented out code, that's what git is for